.github/workflows/builder_docker-based_slsa3.yml

```diff
@@ -363,40 +383,34 @@ jobs:
         uses: ./__BUILDER_CHECKOUT_DIR__/.github/actions/generate-attestations
         with:
           slsa-outputs-file: "${{ needs.build.outputs.slsa-outputs-name }}"
-          predicate-file: "${{ steps.predicate.outputs.predicate }}"
+          predicate-file: "${{ needs.generate-build-definition.outputs.build-definition-name }}"
           output-folder: "attestations-${{ needs.rng.outputs.value }}"
 
       - name: Upload unsigned intoto attestations file for pull request
         if: ${{ github.event_name == 'pull_request' }}
         uses: actions/upload-artifact@83fd05a356d7e2593de66fc9913b3002723633cb # tag=v3.1.1
         with:
           name: "attestations-${{ needs.rng.outputs.value }}"
-          path: "attestations-${{ needs.rng.outputs.value }}"
+          path: "attestations"
```

Comment:
Is this right? I thought this should be the same as `output-folder` on line 387?

Reply:
Ah -- for a pull request event, yes, this is the final output and should be uploaded to a final outputs folder.

I will make the outputs folder an `env` var of the workflow. I'm not so certain that it should be "outputs" hard-coded anyway. Perhaps the workflow outputs should be a string `outputs-folder` that contains either unsigned attestations (for no OIDC token access) or the signed attestations (for real events).

Reply:
Done here as far as placing in the same `output-folder`.